django/shortcuts.py

```diff
@@ -84,16 +116,50 @@ def render(request, template_name, context=None,
                 # warning in RequestContext.__init__.
                 context_instance._current_app = current_app
 
-        content = loader.render_to_string(
+        content = func(
             template_name, context, context_instance, dirs, dictionary,
             using=using)
 
-    return HttpResponse(content, content_type, status)
+    return response_class(content, content_type, status)
+
+
+def render(
+        request, template_name, context=None,
+        context_instance=_context_instance_undefined,
+        content_type=None, status=None, current_app=_current_app_undefined,
+        dirs=_dirs_undefined, dictionary=_dictionary_undefined,
+        using=None):
+    """
+    Return a HttpResponse whose content is filled with the result of calling
+    django.template.loader.render_to_string() with the passed arguments.
+    Uses a RequestContext by default.
+    """
+    return request_context_response_with(loader.render_to_string, HttpResponse,
+                                         request, template_name, context,
+                                         context_instance, content_type, status,
+                                         current_app, dirs, dictionary, using)
+
+
+def stream(
+        request, template_name, context=None,
+        context_instance=_context_instance_undefined,
```

Comment:
Please remove the deprecated parameters from `stream()`: context_instance, current_app, dirs, dictionary. Similarly for `stream_to_response()`.